resources/core/charts/kubeless/values.yaml

```diff
@@ -15,4 +15,4 @@ config:
     secret: ""
 
 tests:
-  enabled: true
+  enabled: false
```

Comment:
is there a reason for disabling the tests?

Reply:
yes:

Reply:
This is the PR i am talking about: https://github.com/kyma-project/kyma/pull/7800